extensions/binding/org.eclipse.smarthome.binding.homematic/README.md

```diff
@@ -0,0 +1,498 @@
+# Homematic Binding
+
+This is the binding for the [eQ-3 Homematic Solution](http://www.eq-3.de/).
+This binding allows you to integrate, view, control and configure all Homematic devices in Eclipse Smarthome.
+
+## Supported Bridges
+
+All gateways which provides the Homematic BIN- or XML-RPC API:
+
+- CCU 1+2
+- [RaspberryMatic](https://github.com/jens-maus/RaspberryMatic)
+- [Homegear](https://www.homegear.eu) (>= 0.8.0-1988)
+- [piVCCU](https://github.com/alexreinert/piVCCU)
+- [YAHM](https://github.com/leonsio/YAHM)
+- [Windows BidCos service](http://www.eq-3.de/downloads.html?kat=download&id=125)
+- [OCCU](https://github.com/eq-3/occu)
+
+The Homematic IP Access Point **does not support** this API and and can't be used with this binding.
+
+Homematic IP support:
+- CCU2 with at least firmware 2.17.15
+- [RaspberryMatic](https://github.com/jens-maus/RaspberryMatic) with the [HM-MOD-RPI-PCB](https://www.elv.de/homematic-funkmodul-fuer-raspberry-pi-bausatz.html) RF module
+- [piVCCU](https://github.com/alexreinert/piVCCU)
+- [YAHM](https://github.com/leonsio/YAHM)
+
+These ports are used by the binding by default to communicate **TO** the gateway:
+
+- RF components: 2001
+- WIRED components: 2000
+- HMIP components: 2010
+- CUxD: 8701
+- TclRegaScript: 8181
+- Groups: 9292
+
+And **FROM** the gateway to the binding:
+
+- XML-RPC: 9125
+- BIN-RPC: 9126
+
+CCU Autodiscovery:
+* UDP 43439
+
+**Note:** The binding tries to identify the gateway with XML-RPC and uses henceforth:
+
+-   **CCU**
+    - **RF**: BIN-RPC
+    - **WIRED**: BIN-RPC
+    - **HMIP**: XML-RPC
+    - **CUxD**: BIN-RPC (CUxD version >= 1.6 required)
+    - **Groups**: XML-RPC
+
+-   **Homegear**
+    - BIN-RPC
+
+-   **Other**
+    - XML-RPC
+
+## Supported Things
+
+All devices connected to a Homematic gateway.
+All required ESH metadata are generated during device discovery.
```

Comment:
Avoid "ESH" as it is no official abbreviation and it should not even turn up in solutions documentation.
Best simply remove ESH here.